### PR TITLE
Add cls training

### PR DESCRIPTION
**Pull request by rentainhe**
## TODO
在projects文件夹下适配一套训练代码，整体参考swin-t
- [x] scheduler
- [x] optim
- [x] logger
- [x] data
- [x] train
- [x] model builder
- [x] bash files

### CODE REVIEW

File: projects/classification/config.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+# --------------------------------------------------------
+# Swin Transformer
+# Copyright (c) 2021 Microsoft
+# Licensed under The MIT License [see LICENSE for details]
+# Written by Ze Liu
+# --------------------------------------------------------'
```
Comment (simonJJJ) [inline projects/classification/config.py:6]: 这个License要改下，可以写modifiled from ... 或者based on ...

File: projects/classification/data/build.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# --------------------------------------------------------
+# Swin Transformer
+# Copyright (c) 2021 Microsoft
+# Licensed under The MIT License [see LICENSE for details]
+# Written by Ze Liu
+# --------------------------------------------------------
```
Comment (simonJJJ) [inline projects/classification/data/build.py:6]: 同上

File: projects/classification/data/cached_image_folder.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
+# --------------------------------------------------------
+# Swin Transformer
+# Copyright (c) 2021 Microsoft
+# Licensed under The MIT License [see LICENSE for details]
+# Written by Ze Liu
+# --------------------------------------------------------
```
Comment (simonJJJ) [inline projects/classification/data/cached_image_folder.py:6]: 同上

File: projects/classification/main.py [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+@flow.no_grad()
+def throughput(data_loader, model, logger):
+    model.eval()
+
+    for idx, (images, _) in enumerate(data_loader):
+        images = images.cuda()
+        batch_size = images.shape[0]
+        for i in range(50):
+            model(images)
+        # flow.cuda.synchronize()
```
Comment (simonJJJ) [inline projects/classification/main.py:341]: 用不到的就删掉

File: projects/classification/config.py (hunk 1 above)
Comment (rentainhe) [reply projects/classification/config.py:6]: OK，忘记了

**Review by simonJJJ — APPROVED** [review]
